docs/docusaurus.config.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
       {
         docs: {
           // It is recommended to set document id as docs home page (`docs/` path).
-          homePageId: "docs/gettingStarted",
```

Comment:
this just fixes a warning i was getting from docusaurus - it's preferred to use slug to denote the homepage